### PR TITLE
UI Branding Updates

### PR DESCRIPTION
**Pull request by Ryan-Cleminson**

### CODE REVIEW

**Comment by Ryan-Cleminson** [comment]
@albert118 please review my UI updates and changes in the branch

**Review by albert118 — COMMENTED** [review]
looks like you're missing a dependency change for `pacakge.json`, I can see `react-bootstrap` being pulled in

File: src/frontend/src/styles/pages/Dashboard.scss [hunk 1]
```diff
@@ -30,28 +39,16 @@
...
-}
-
-// .dashboard-page__heading {
-//     @include type-style('productive-heading-05');
-// }
-
-.dashboard-page__subheading {
-    // @include type-style('productive-heading-03');
-    font-weight: 600;
+    height: auto;
```
Comment (albert118) [inline src/frontend/src/styles/pages/Dashboard.scss:42]: we only need to set the width to 100% with the simplified structure (ie. remove max-width and auto height). Good job simplifying this!

```suggestion
    width: 100%
```

File: src/frontend/src/styles/pages/Dashboard.scss [hunk 2]
```diff
@@ -2,26 +2,35 @@
...
-    margin-left: -20px;
-    margin-right: -20px;
-    @include breakpoint-down(md) {
-        margin-left: 0;
-        margin-right: 0;
-    }
+.dashboard-page{
+    min-height: 70vh;
+    padding: $spacing-08;
+    background: linear-gradient(.60turn, #0f4beb,transparent);
```
Comment (albert118) [inline src/frontend/src/styles/pages/Dashboard.scss:8]: this is great, and I was thinking something similar. But think bigger - let's get this in more places, on more pages, and keep it consistent. I've already introduced something that can set this pattern up in a theme

https://github.com/albert118/Studio-Management-System/pull/106/files#diff-8cd8865459732fa8f273ace6c05385879c19c0aa4e8aecdc3312f9dacd9343e7R30-R36

Comment (albert118) [reply src/frontend/src/styles/pages/Dashboard.scss:8]: It'll be easier to follow up in a second PR to do all the backgrounds though, 😄

**Review by albert118 — COMMENTED** [review]
Overall looking great to clean lots of this up. I've left some comments for minor touch-ups before we merge this in

**Comment by albert118** [comment]
I did notice that we're not consuming 100% of the viewport height, notice the white-gap at the bottom of the page

image

**Comment by albert118** [comment]
> I did notice that we're not consuming 100% of the viewport height, notice the white gap at the bottom of the page

The min-height has helped correct this issue, but we should address on the `main` block, rather than on a per grid/page basis,

image

If we tweak the layout of the `main` and `footer` blocks, we can get closer to this,

image

This requires that the blocks (`main` and `footer`),

* set min-heights (70vh and 30vh respectively)
* set their backgrounds - not their internal wrappers
* set padding for their internal element layouts (a block should pad it's child elements)

File: src/frontend/src/styles/pages/Dashboard.scss (hunk 2 above)
Comment (Ryan-Cleminson) [reply src/frontend/src/styles/pages/Dashboard.scss:8]: Yep good idea, personally think the home page should look a little different to the other pages as those a more for function. That said I think we can leave that for next semester.

**Comment by Ryan-Cleminson** [comment]
@albert118 just pushed some changes that hopefully address what you asked for :)

**Review by albert118 — APPROVED** [review]

**Pull request merged**